util/util.go

```diff
@@ -236,6 +236,10 @@ func Now() string {
 	return time.Now().UTC().Format(time.RFC3339)
 }
 
+func DurationAfterNow(d time.Duration) string {
```

Comment:
I don't think this name makes sense. `DurationAfterNow` seems like it would return a duration instead of a UTC formatted timestamp.

Should it be `TimestampAfterDuration`?

Reply:
Thank you. Fixed it